.github/workflows/mavenCI.yaml

```
@@ -0,0 +1,28 @@
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

The feature branches will branched and merged to "develop"
```suggestion
    branches: [ "develop" ]
```